src/UniOneClient.php

```diff
@@ -43,25 +43,29 @@ final class UniOneClient
     private string $apiKey;
 
     /**
-     * @param string $endpoint
      * @param string $apiKey
+     * @param string $endpoint
      * @param array  $config
      */
-    public function __construct(string $endpoint, string $apiKey, array $config = [])
+    public function __construct(string $apiKey, string $endpoint = '', array $config = [])
     {
-        $this->setEndpoint($endpoint)->setApiKey($apiKey);
-        $defaults = [
+        $this->setApiKey($apiKey);
+        if (!empty(\trim($endpoint))) {
```

Comment:
Please remove trim(). We should encourage people to pass correct data, and we won't be able to handle everything.